src/black/linegen.py

```diff
@@ -197,6 +197,26 @@ def visit_decorators(self, node: Node) -> Iterator[Line]:
             yield from self.line()
             yield from self.visit(child)
 
+    def visit_power(self, node: Node) -> Iterator[Line]:
+        for idx, leaf in enumerate(node.children[:-1]):
+            next_leaf = node.children[idx + 1]
+
+            if not isinstance(leaf, Leaf):
+                continue
+
+            value = leaf.value.lower()
+            if (
+                leaf.type == token.NUMBER
+                and next_leaf.type == syms.trailer
+                # It shouldn't wrap hexadecimal, binary and octal literals
+                and (not value.startswith(("0x", "0b", "0o")))
```

Comment:
```suggestion
                and not value.startswith(("0x", "0b", "0o"))
```
NIT: let's not have the extra parens

Reply:
I find the brackets to be more readable than not having them.

Reply:
I think it would make sense to follow the existing convention as @felix-hilden just said, e.g:
```console
$ grep -rni 'and (not' src/black
$ grep -rni 'and not' src/black
src/black/trans.py:346:            and not has_triple_quotes(string_leaf.value)
src/black/trans.py:682:                and not is_empty_rpar(LL[next_idx])
src/black/trans.py:1228:                and not csplit.has_prefix
src/black/trans.py:1416:                and not breaks_unsplittable_expression(i)
src/black/trans.py:1421:        while is_valid_index(break_idx - 1) and not passes_all_checks(break_idx):
src/black/trans.py:1431:            while is_valid_index(break_idx + 1) and not passes_all_checks(break_idx):
src/black/trans.py:1458:        if "f" in prefix and not fstring_contains_expr(string):
src/black/__init__.py:92:        if check and not diff:
src/black/__init__.py:139:    if target_version is not None and not isinstance(target_version, list):
src/black/__init__.py:586:            if p.suffix == ".ipynb" and not jupyter_dependencies_are_installed(
src/black/__init__.py:971:    if not fast and not mode.is_ipynb:
src/black/__init__.py:1200:            if len(n.children) > 1 and not is_simple_decorator_expression(
src/black/linegen.py:332:        and not line.should_split_rhs
src/black/linegen.py:333:        and not line.magic_trailing_comma
src/black/linegen.py:338:        and not (line.inside_brackets and line.contains_standalone_comments())
[...]
```